src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -150,12 +153,12 @@
       </_SupplementalTestData>
     </ItemGroup>
     <ItemGroup>
-      <SupplementalTestDataTestDir Include="@(_SupplementalTestData->'$(TestPath)/%(DestinationDir)%(Filename)%(Extension)')" />
```

Comment:
I would still like to eliminate it and I don't think the hard linking should be a blocker, we shouldn't be in a position where we need that as a feature it should only be a optimization for all copying or not. I'm fine doing that as a separate change but please add a tracking issue.